src/metabase/public_settings.clj

```diff
@@ -828,6 +828,7 @@ See [fonts](../configuring-metabase/fonts.md).")
                       :query_reference_validation     (premium-features/enable-query-reference-validation?)
                       :sandboxes                      (premium-features/enable-sandboxes?)
                       :scim                           (premium-features/enable-scim?)
+                      :serialization                  (premium-features/enable-serialization?)
                       :session_timeout_config         (premium-features/enable-session-timeout-config?)
```

Comment:
apparently this map is actually missing a bunch of features that are actually in the tokens 😫

Reply:
that issue is captured https://github.com/metabase/metabase/issues/48355. thanks for pointing it out @iethree !